client/fingerprint_manager.go

```diff
@@ -2,6 +2,7 @@ package client
 
 import (
 	"log"
+	"strings"
```

Comment:
Can you try to arrange the methods such that drivers methods are colocated and fingerprint methods are too. I think it would be simpler to follow if it went:

1. Struct
2. NewFingerprintManager
3. Run.
4. SetupFingerprinter
5. SetupDriver
6-X: Fingerprinter stuff
7-X: Driver stuff